server/src/test/java/io/spine/server/QueryServiceTest.java

```diff
@@ -102,7 +104,7 @@ void executeQueries() {
     }
 
     @Test
-    @DisplayName("dispatch queries to proper bounded context")
+    @DisplayName("dispatch queries to proper Bounded Context")
```

Comment:
Why do we capitalize "Bounded Context"? If this refers to the Java class rather than a concept, please join the name into a single word.

Reply:
After Fowler :) https://martinfowler.com/bliki/BoundedContext.html